lib/instrumentation/redis.js

```diff
@@ -12,8 +12,23 @@ const {
 } = require('../shim/specs')
 
 module.exports = function initialize(_agent, redis, _moduleName, shim) {
+  const version = Number(shim.pkgVersion.split('.').shift())
```

Comment:
we talked and agreed none of this is needed aside from the `shim.logger.warn` on line 31 as the instrumentation for v4+ is int `lib/instrumentation/@node-redis/client.js`